src/core/DrumSynth.cpp

```diff
@@ -192,7 +192,7 @@ int DrumSynth::GetPrivateProfileString(
 	bool inSection = false;
 	int len = 0;
 	constexpr auto linelen = 200;
-	std::array<char, linelen> line = {};
+	char line[linelen] = {};
```

Comment:
We've been making an effort to use `std::array` instead of C arrays (see #6564).

You can use its `data()` member to get a pointer to the first element since it doesn't decay to a pointer like C arrays do.

Reply:
> We've been making an effort to use std::array instead of C arrays (see https://github.com/LMMS/lmms/pull/6564).

Indeed: https://github.com/LMMS/lmms/pull/6564#issuecomment-1307944978

I've switched it to a `std::array`. There is only one place where it is indexed into, the rest is just calls to `.data()` in order to pass it into functions as a `char*`. I don't think this is an improvement in this context.